src/Student.Email.Assistant.Backend/features/generate_title/generate_title_endpoint.py

```diff
@@ -1,8 +1,14 @@
 from fastapi import APIRouter
 from features.generate_title.request import GenerateTitleRequest
+import infrastructure.openai_client
 
 router = APIRouter()
 
 @router.post("/api/generate-title")
 async def generate_title(request: GenerateTitleRequest) -> str:
-    return request.email_content
\ No newline at end of file
+    prompt = f"Please generate an email title for this email:{request.email.content}.
+                Email is written in an academic enviroment so be sure to use appropriate titles.
+                Be polite and make sure that title is as informative as possible.
+                Respond in language email was written in, only with the title."
+    response = infrastructure.openai_client.call_openai_api(prompt)
```

Comment:
to co przekazujesz jako prompt to powinna być tablica z obiektami które informują jaka część jest od systemu, a jaka od usera. Przykład jest w **openai_client.py**

W aktualnej formie ten kod nie działa (sprawdziłem sobie lokalnie)

Reply:
Poprawiłem w innym PR